core/src/main/java/greencity/entity/SocialNetwork.java

```diff
@@ -0,0 +1,28 @@
+package greencity.entity;
+
+import javax.persistence.*;
+import lombok.*;
+import org.hibernate.validator.constraints.Length;
+
+@Entity
+@NoArgsConstructor
+@AllArgsConstructor
+@Getter
+@Setter
+@Builder
+@Table(name = "social_networks")
+public class SocialNetwork {
+    @Id
+    @GeneratedValue(strategy = GenerationType.IDENTITY)
+    private Long id;
+
+    @Length(min = 1, max = 500)
+    @Column(name = "social_network_url", length = 500)
+    String url;
+
+    @ManyToOne(fetch = FetchType.LAZY)
```

Comment:
I don't think it should be LAZY from this side of the connection.

Reply:
What does that mean?